src/containers/Network/css/barchart.scss

```diff
@@ -0,0 +1,13 @@
+p {
```

Comment:
This will change all `<p>` tags globally.  You should make `p` and `.yLabel` styles use specific selectors.